src/pages/DigitalAssets/routes/AddAsset/AddAssetContainer.js

```diff
@@ -12,8 +12,14 @@ import {
   submitAssetForm,
 } from 'store/modules/addAsset'
 
+// eslint-disable-next-line import/no-duplicates
 import AddAssetView from './AddAssetView'
 
+// eslint-disable-next-line import/no-duplicates
+import { type Props } from './AddAssetView'
```

Comment:
why do we have two separate imports from the same file?

Reply:
There was the issue, when our storybook was broken, when we are using something, like this
`import AddAssetView, { type Props } from './AddAssetView'`

This is old code, in future it will be refactored
I decide don't touch component exports here (ie `export default AddAssetView` -> `export {AddAssetView}` ), because this task about flow types and redux connects only